src/components/PasswordChecker/PasswordChecker.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react';
+import PasswordInput from './PasswordInput';
+import StrengthIndicator from './StrengthIndicator';
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+import Typography from '@mui/material/Typography';
+
+const PasswordChecker: React.FC = () => {
+  const [password, setPassword] = useState<string>('');
+  const [timeToCrack, setTimeToCrack] = useState<string>('0 seconds');
+  const [bgColor, setBgColor] = useState<string>('blue');
+
+  const calculateStrength = (password: string): void => {
+    if (password.length === 0) {
+      setTimeToCrack('0 seconds');
+      setBgColor('blue');
+      return;
+    }
+
+    const length = password.length;
+    let characterSetSize = 0;
+
+    if (/[a-z]/.test(password)) characterSetSize += 26;
+    if (/[A-Z]/.test(password)) characterSetSize += 26;
+    if (/\d/.test(password)) characterSetSize += 10;
+    if (/[\W_]/.test(password)) characterSetSize += 32;
+
```

Comment:
```suggestion
	const getCharacterSetSize = (password: string): number => {
		let size = 0
		if (/[a-z]/.test(password)) size += 26
		if (/[A-Z]/.test(password)) size += 26
		if (/\d/.test(password)) size += 10
		if (/[\W_]/.test(password)) size += 32
		return size
	}
```
This can be changed to a helper function and put into a helper folder like so, src/features/passwordChecker/helpers.ts